norms.md

```diff
@@ -0,0 +1,22 @@
+##Fishtown Analytics Norms##
```

Comment:
I need your help understanding what the difference between a norm and a value is. It seems like you're heavily borrowing from what I had listed as values, so it seems like there is some overlap in our conceptions.

I'm also working on something I'm calling the "company operating system" which I had imagined overlapping with norms. It's a bit more specific than simply a list, however.

In general, I just think we should define the structure of this activity when we get together. I'm thinking through it as I go (as I'm sure are you) so I'm sure it's all productive, we just need to figure out how we're getting to some type of state we're happy with.